src/coreComponents/constitutive/ConstitutiveManager.cpp

```diff
@@ -41,12 +41,14 @@ ConstitutiveManager::~ConstitutiveManager()
 {}
 
 
-Group * ConstitutiveManager::createChild( string const & childKey, string const & childName )
+Group * ConstitutiveManager::createChild( string const & childKey,
+                                          string const & childName,
+                                          bool const allowExistence )
```

Comment:
There were some places where `createChild` needed to change, since this is a virtual function in Group, changing the signature there required changes everywhere.